test/cypress/e2e/share-prompt.cy.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable cypress/no-unnecessary-waiting */
+describe('Sharing a Prompt', () => {
+  it('Should display properly and navigate to /month page', () => {
+    // Visits the month page and waits for 5 seconds for the page to load
+    cy.visit('/month').wait(5000)
```

Comment:
Wait is not a good practice. 

If you need to wait someting will appear you can use  timeout instead


### Ressources 

[Wait Anti patern](https://docs.cypress.io/api/commands/wait#Time)

[Timeout](https://docs.cypress.io/guides/core-concepts/retry-ability#Timeouts)

Reply:
I see that you approved changes. I'll approve now to and merge. We're all learning Cypress so we can circle round and get this anti-pattern later. Personally, Cypress is top on my list next week.

Reply:
Yes it's why i approve.

Reply:
OH, good to know!
Let's change it in `develop` branch then.